src/pyOpenMS/tests/unittests/test_dataframe_columns.py

```
@@ -1146,8 +1146,9 @@ def test_featuremap_missing_spectrum_native_id(self):
         assert len(df) == 2
         # First feature should have the native_id
         assert df.iloc[0]['ID_native_id'] == 'scan=100'
-        # Second feature should have 'None' string (numpy converts None to string due to U100 dtype)
-        assert df.iloc[1]['ID_native_id'] == 'None'
+        # Second feature should have NaN (spectrum_native_id meta value not set)
+        # pd.DataFrame converts None to NaN for object dtype columns
+        assert pd.isna(df.iloc[1]['ID_native_id'])
```

Ruff (0.14.14)

[error] Undefined name `pd`

(F821)